activation/poetdb.go

```diff
@@ -127,16 +136,20 @@ func (db *PoetDb) publishPoetProofRef(key poetProofKey, poetProofRef []byte) {
 	delete(db.poetProofRefSubscriptions, key)
 }
 
-func (db *PoetDb) GetMembershipByPoetProofRef(poetRef []byte) (map[common.Hash]bool, error) {
-	poetProofBytes, err := db.store.Get(poetRef)
+func (db *PoetDb) GetPoetProofMessage(proofRef []byte) ([]byte, error) {
```

Comment:
ou can shorten this to Get the poet part is implied
also you should use this method inside GetMembershipByPoetProofRef
i would change "GetMembershipByPoetProofRef" to GetAsMessage or something that implies the conversion from bytes